src/db.ls

```diff
@@ -67,16 +67,61 @@
     db.DB = {}
     minimatch = require \minimatch
     try
-      db.DB = JSON.parse do
-        require \fs .readFileSync "#dataDir/dump.json" \utf8
-      console.log "==> Restored previous session from JSON file"
+      if fs.existsSync "#dataDir/dump/"
+        fs.readdirSync("#dataDir/dump/").forEach (f) ->
+          key = f.split(".")[0]
+          type = key.split("-")[0]
+          if type is "timestamps"
+            db.DB.timestamps = JSON.parse(fs.readFileSync("#dataDir/dump/timestamps.json", \utf8))
+          else if type is "snapshot"
+            db.DB[key] = fs.readFileSync("#dataDir/dump/#key.txt", \utf8)
+          else if type is "audit"
+            db.DB[key] = fs.readFileSync("#dataDir/dump/#key.txt", \utf8).split("\n")
+            for k, v of db.DB[key]
+              db.DB[key][k] = db.DB[key][k].replace(/\\n/g,"\n").replace(/\\r/g,"\r").replace(/\\\\/g,"\\")
+      else
+        db.DB = JSON.parse(fs.readFileSync "#dataDir/dump.json" \utf8)
+
+      console.log "==> Restored previous session from dump storage"
       db.DB = {} if db.DB is true
     Commands =
       bgsave: (cb) ->
+        if !fs.existsSync "#dataDir/dump/"
+          fs.mkdirSync "#dataDir/dump/"
+
+        oldTimestamps = {}
+        if !fs.existsSync "#dataDir/dump/timestamps.json"
+          for k, v of db.DB.timestamps
+            id = k.split("-").pop!
+            oldTimestamps[id] = 0 #if no timestamp file is found, pretend the sheets haven't been saved in litteral decades
```

Comment:
`litteral` => `literally`

Otherwise +1 from me — LGTM — thanks for the contribution!